findings_manager.tf

```
@@ -22,6 +22,12 @@ data "aws_iam_policy_document" "findings_manager_lambda_iam_role" {
     resources = ["${module.findings_manager_bucket.arn}/*"]
   }
 
+  statement {
+    sid       = "S3ListBucketObjects"
+    actions   = ["s3:ListBucket"]
+    resources = ["${module.findings_manager_bucket.arn}/*"]
```

Doesn't ListBucket need a bucket resource instead of the wildcard object resource?
```suggestion
    resources = ["${module.findings_manager_bucket.arn}"]
```